.github/workflows/publish-antora-docs.yml

```diff
@@ -22,50 +22,75 @@ permissions:
 name: Antora Docs
 
 jobs:
-  docs_skip:
-    name: Docs Skip Duplicates
-    continue-on-error: true # Uncomment once integration is finished
-    runs-on: ubuntu-latest
-    # Map a step output to a job output
-    outputs:
-      should_skip: ${{ steps.skip_check.outputs.should_skip }}
-      run_id: ${{ steps.skip_check.outputs.skipped_by.id }}
-    steps:
-      - id: skip_check
-        uses: fkirc/skip-duplicate-actions@v5
-        with:
-          concurrent_skipping: "never"
-          skip_after_successful_duplicate: "false"
-          paths: '["docs-src/**", ".github/workflows/publish-antora-docs.yml"]'
-          do_not_skip: '["workflow_dispatch", "schedule"]'
+  # docs_skip:
+  #   name: Docs Skip Duplicates
+  #   continue-on-error: true # Uncomment once integration is finished
+  #   runs-on: ubuntu-latest
+  #   # Map a step output to a job output
+  #   outputs:
+  #     should_skip: ${{ steps.skip_check.outputs.should_skip }}
+  #     run_id: ${{ steps.skip_check.outputs.skipped_by.id }}
+  #   steps:
+  #     - id: skip_check
+  #       uses: fkirc/skip-duplicate-actions@v5
+  #       with:
+  #         concurrent_skipping: "never"
+  #         skip_after_successful_duplicate: "false"
+  #         paths: '["docs-src/**", ".github/workflows/publish-antora-docs.yml"]'
+  #         do_not_skip: '["workflow_dispatch", "schedule"]'
```

Comment:
do you want to un-comment this stuff?  and others in this PR?

Reply:
Yes